certified-connectors/Iconfirm/apiProperties.json

```diff
@@ -0,0 +1,22 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "api_key": {
+        "type": "securestring",
+        "uiDefinition": {
+          "displayName": "API-key",
+          "description": "Iconfirm API-key for use with the connector",
+          "tooltip": "Input your API-key generated for an API user on your Iconfirm subscription",
+          "constraints": {
+            "tabIndex": 2,
+            "clearText": false,
+            "required": "true"
+          }
+        }
+      }
+    },
+    "iconBrandColor": "#EDECEC",
+    "capabilities": [],
+    "publisher": "Nicolai Berg"
```

Comment:
@iconfirm-development The publisher value seems an individual name that is not allowed for a certified connector. If you want to publish this connector with your name, you need to place your connector under "independent-publisher connectors" folder. If it is a typo, you can update it with your organization's name.

And you have missed adding `"stackOwner"` property with is required one.

Docs - [Add metadata](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#step-3a-publisher-and-stackowner-properties)

Reply:
The publisher has been removed now

Reply:
@iconfirm-development The `"publisher"` and `"stackOwner" properties are required.